### PR TITLE
Merge 2.03 development to 2.03 live branch

### PR DESCRIPTION
**Pull request by dalepotter**
Implements version 2.03 ruleset changes based on the [final technical proposal](https://discuss.iatistandard.org/t/final-technical-proposal-for-the-2-03-upgrade/1097).

### CODE REVIEW

**Comment by andylolz** [comment]
I’ve added a comment to a related issue: https://github.com/IATI/IATI-Rulesets/issues/50#issuecomment-357007837

Referencing it here, as per [the inspection and testing discuss post](https://discuss.iatistandard.org/t/version-2-03-available-for-inspection-and-testing/1176).

File: rulesets/standard.json [hunk 1]
```diff
@@ -57,6 +57,14 @@
             ]
         }
     },
+    "//planned-disbursement/provider-org | //planned-disbursement/receiver-org | //transaction/provider-org | //transaction/receiver-org | //recipient-org-budget/recipient-org": {
+        "regex_matches": {
+            "cases": [
+                { "regex": "^(?!\s*$).+",
```
Comment (dalepotter) [inline rulesets/standard.json]: It appears that this is not a valid JSON string:

```
>>> json.loads(r'{ "regex": "^(?!\s*$).+"}')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/json/__init__.py", line 326, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Invalid \escape: line 1 column 16 (char 16)
```

Comment (dalepotter) [reply rulesets/standard.json]: It appears the fix to this issue is to only reference a subset of regex syntax described here: https://spacetelescope.github.io/understanding-json-schema/reference/regular_expressions.html#regular-expressions

**Comment by dalepotter** [comment]
Regex needs fixing, so that the ruleset file contains valid json.

**Comment by hayfield** [comment]
There are a couple of oddities with the display of the new Rules, though this has been discussed and deemed OK for deployment.

**Pull request merged**